Jenkinsfile

```diff
@@ -1,15 +1,15 @@
 buildPlugin(configurations: [
     [platform: 'linux', jdk: '11'],
     [platform: 'windows', jdk: '11'],
-])
+], useContainerAgent: false)
```

Comment:
currently the default, but the container agents don't have docker so lets be specific rather than get some failures if the default changes.